contracts/Controller.sol

```diff
@@ -0,0 +1,14 @@
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ */
+pragma solidity 0.6.10;
+
+import {MarginCalculator} from "./MarginCalculator.sol";
+
+/**
+ *
+ */
+// solhint-disable-next-line no-empty-blocks
+contract Controller is MarginCalculator {
```

Comment:
Aren't we separating these two into two contracts?

Reply:
Ah wasn't sure if we doing that or we writing everything in the same contract.